src/api/modules/utils.ts

```diff
@@ -129,81 +128,94 @@ export const appIdFromRegistry = (app: string, majorLocator: string) => {
     .catch(handleError(app))
 }
 
-export function optionsFormatter(billingOptions: BillingOptions) {
+const FREE_BILLING_OPTIONS_TYPE = 'free'
+export const isFreeApp = (type?: string, free?: boolean) => type === FREE_BILLING_OPTIONS_TYPE || free
+
+type BillingInfo = {
```

Comment:
If this type will be used by more files like `BillingOptions`, maybe is a good idea to move to `/src/typings/global.d.ts`

Reply:
Currently only used here